common/src/main/java/com/wynntils/core/features/overlays/BasicOverlay.java

```diff
@@ -6,22 +6,31 @@
 
 import com.mojang.blaze3d.platform.Window;
 import com.mojang.blaze3d.vertex.PoseStack;
-import com.wynntils.utils.objects.PentaConsumer;
+import com.wynntils.core.features.overlays.sizes.OverlaySize;
+import com.wynntils.utils.objects.QuadConsumer;
 
 public class BasicOverlay extends Overlay {
-    private final PentaConsumer<Overlay, OverlayPosition, PoseStack, Float, Window> renderConsumer;
+    private final QuadConsumer<Overlay, PoseStack, Float, Window> renderConsumer;
```

Comment:
It might be good to just have this a functional interface, we probably won't use QuadConsumers in many places.